src/Client.re

```diff
@@ -1,4 +1,10 @@
-type t;
+type t = {
+  url: string,
+  suspense: bool,
+  preferGetMethod: bool,
+  requestPolicy: string,
+  maskTypename: bool,
+};
```

Comment:
We only expose _some_ properties on `Client` publicly (not the methods). `url` is the one that I can see being most commonly sought, and we use it internally to generate safe fallback urls.